File: src/main/java/space/accident/api/interfaces/tileentity/IColoredTileEntity.java
package space.accident.api.interfaces.tileentity;

public interface IColoredTileEntity {
	/**
	 * @return 0 - 15 are Colors, while -1 means uncolored
	 */
	int getColorization();
	
	/**
	 * Sets the Color Modulation of the Block
	 *
	 * @param color the Color you want to set it to. -1 for reset.
	 */
	int setColorization(int color);
}

File: src/main/kotlin/space/accident/api/interfaces/te/IColoredTileEntity.kt
package space.accident.api.interfaces.te

interface IColoredTileEntity {
    /**
     * @return 0 - 15 are Colors, while -1 means uncolored
     */
    fun getColorization(): Byte

    /**
     * Sets the Color Modulation of the Block
     *
     * @param aColor the Color you want to set it to. -1 for reset.
     */
    fun setColorization(aColor: Byte): Byte
}
